web/src/containers/loans/Loan.js

```diff
@@ -87,7 +96,13 @@ function Loan(props) {
     console.log("user:", Info.user);
 
     const account = walletService.checkAccount();
-    if (loan && loan.state.toUpperCase() == "FUNDING" && account != null) {
+    if (
+      loan &&
+      loan.state.toUpperCase() != "PENDING" &&
+      loan.state.toUpperCase() != "WITHDRAWN" &&
+      loan.state.toUpperCase() != "REJECTED" &&
+      account != null
+    ) {
```

Comment:
treat each state as a component, cf. LoanActive. Cleaner code as it has no conditionals